packages/web3-core/src/web3_subscription_manager.ts

```diff
@@ -75,10 +93,19 @@ export class Web3SubscriptionManager<
 		return subscription;
 	}
 
+	/**
+	 * Will returns all subscriptions.
+	 */
 	public get subscriptions() {
 		return this._subscriptions;
 	}
 
+	/**
+	 *
+	 * @param sub A {@link Web3Subscription} objects
+	 *
+	 * Adds an instance of {@link Web3Subscription} and subscribes to it
```

Comment:
```suggestion
	 * Adds an instance of {@link Web3Subscription} and subscribes to it
	 *
	 * @param sub - A {@link Web3Subscription} objects
```
We need to put the description and then the list of the `@param`s.